### PR TITLE
Add `outputPath` option to `ResolveTagHelpersCommand`.

### PR DESCRIPTION
**Pull request by NTaylorMullen**
#48

### CODE REVIEW

File: src/dotnet-razor-tooling/Internal/ResolveTagHelpersCommand.cs [hunk 1]
```diff
@@ -28,6 +28,10 @@ internal static void Register(CommandLineApplication app)
                     "-p|--protocol",
                     "Protocol to resolve TagHelperDescriptors with.",
                     CommandOptionType.SingleValue);
+                var outputPath = config.Option(
+                    "-o|--outputPath",
+                    "The projects output path.",
```
Comment (Eilon) [inline src/dotnet-razor-tooling/Internal/ResolveTagHelpersCommand.cs]: `project's` (possessive)

Comment (Eilon) [inline src/dotnet-razor-tooling/Internal/ResolveTagHelpersCommand.cs]: Or say `The output path of the project`

Comment (Eilon) [inline src/dotnet-razor-tooling/Internal/ResolveTagHelpersCommand.cs]: Or `The path to which the output of the project will go.`

Comment (Eilon) [inline src/dotnet-razor-tooling/Internal/ResolveTagHelpersCommand.cs]: :trollface:

Comment (NTaylorMullen) [inline src/dotnet-razor-tooling/Internal/ResolveTagHelpersCommand.cs]: Got it. So it's going to be `The path to which the output of the project will go. :trollface:`.

Perfect verbage

Comment (Eilon) [inline src/dotnet-razor-tooling/Internal/ResolveTagHelpersCommand.cs]: Yeah sure :smile:

Comment (dougbu) [inline src/dotnet-razor-tooling/Internal/ResolveTagHelpersCommand.cs]: verbiage :trollface:

**Comment by bricelam** [comment]
Colloquially, we call it the bin directory. Maybe `--bin-dir`?

**Comment by NTaylorMullen** [comment]
Hmm, do we typically not pascal case our command line options? If not, `-b|--binDirectory`?

**Comment by bricelam** [comment]
No; kabob-case. @davidfowl yelled at me for trying to pascal here.

**Comment by NTaylorMullen** [comment]
Hah, alright. `-b|--bin-dir` it is.

**Comment by moozzyk** [comment]
@NTaylorMullen - publish-iis has `-p` because it needs the output of the publish command and not the the output of the build command. But it would be good to review all the parameters and unify them.

**Pull request closed without merging**